modules/graphdb/user_data.tf

```diff
@@ -127,7 +125,6 @@ data "cloudinit_config" "entrypoint" {
       content = templatefile("${path.module}/templates/08_cluster_setup.sh.tpl", {
         app_configuration_endpoint : var.app_configuration_endpoint
         private_dns_zone_name : azurerm_private_dns_zone.graphdb.name
-        node_count : var.node_count
```

Comment:
`# 10 Start GDB services - Single node` also refers to this node_count but I don't see it int he script